internal/cli/users_blocks.go

```diff
@@ -0,0 +1,102 @@
+package cli
+
+import (
+	"fmt"
+
+	"github.com/auth0/go-auth0/management"
+	"github.com/spf13/cobra"
+
+	"github.com/auth0/auth0-cli/internal/ansi"
+)
+
+func userBlocksCmd(cli *cli) *cobra.Command {
+	cmd := &cobra.Command{
+		Use:   "blocks",
+		Short: "Manage brute-force protection user blocks",
+		Long:  "Manage brute-force protection user blocks.",
+	}
+
+	cmd.SetUsageTemplate(resourceUsageTemplate())
+	cmd.AddCommand(listUserBlocksCmd(cli))
+	cmd.AddCommand(deleteUserBlocksCmd(cli))
+
+	return cmd
+}
+
+func listUserBlocksCmd(cli *cli) *cobra.Command {
+	var inputs struct {
+		userID string
+	}
+
+	cmd := &cobra.Command{
+		Use:   "list",
+		Args:  cobra.MaximumNArgs(1),
+		Short: "List brute-force protection blocks for a given user",
+		Long:  "List brute-force protection blocks for a given user.",
+		Example: `  auth0 users blocks list <user-id>
+  auth0 users blocks list <user-id> --json`,
+		RunE: func(cmd *cobra.Command, args []string) error {
+			if len(args) == 0 {
+				if err := userID.Ask(cmd, &inputs.userID); err != nil {
+					return err
+				}
+			} else {
+				inputs.userID = args[0]
+			}
+
+			var userBlocks []*management.UserBlock
+
+			err := ansi.Waiting(func() error {
+				var err error
+				userBlocks, err = cli.api.User.Blocks(inputs.userID)
+				return err
+			})
+
+			if err != nil {
+				return fmt.Errorf("Unable to load user blocks %v, error: %w", inputs.userID, err)
+			}
+
+			cli.renderer.UserBlocksList(userBlocks)
+			return nil
+		},
+	}
+
+	cmd.Flags().BoolVar(&cli.json, "json", false, "Output in json format.")
+
+	return cmd
+}
+
+func deleteUserBlocksCmd(cli *cli) *cobra.Command {
+	var inputs struct {
+		userID string
+	}
+
+	cmd := &cobra.Command{
+		Use:     "unblock",
+		Args:    cobra.MaximumNArgs(1),
+		Short:   "Remove brute-force protection blocks for a given user",
+		Long:    "Remove brute-force protection blocks for a given user.",
+		Example: `  auth0 users unblock <user-id>`,
```

Comment:
```suggestion
		Example: `  auth0 users blocks unblock <user-id>`,
```

Reply:
Fixed in [`c125485` (#617)](https://github.com/auth0/auth0-cli/pull/617/commits/c125485b92f33f939d136a92fd71dacd65ba9b70)